lib/Mock/mocks/NativeCommandsSender.tsx

```diff
@@ -47,6 +47,7 @@ export class NativeCommandsSender {
 
   pop(commandId: string, componentId: string, _options?: object) {
     return new Promise((resolve) => {
+      if (!LayoutStore.getLayoutById(componentId)) return;
```

Comment:
curious, what's the use-case here? Why does this return silently? (should we consider throwing an error/warning instead?

Reply:
Fixed 👍